build/ci.go

```diff
@@ -942,6 +942,8 @@ func doXCodeFramework(cmdline []string) {
 
 	// Build the iOS XCode framework
 	build.MustRun(goTool("get", "golang.org/x/mobile/cmd/gomobile", "golang.org/x/mobile/cmd/gobind"))
+	// Patch gomobile to disable bitcode for now (rust generated bls lib output is not compatible)
+	build.MustRunCommand("sed", "-i", "", `/^[[:space:]]*cflags += \" -fembed-bitcode\"$/s/^/\/\//`, filepath.Join(build.GOPATH(), "src/golang.org/x/mobile/cmd/gomobile/env.go"))
```

Comment:
Too bad it doesn't work :(
Can you try with Rust 1.37? I'm upgrading the build toolchain to that anyway. Seems like people succeeded with recent nightlies, so it might work there: https://github.com/rust-lang/rust/issues/35968#issuecomment-529618704

Reply:
I tried with Rust 1.38 stable and 1.40.0-nightly (2748a9fd9 2019-10-08).
Both failed to link with gomobile when bitcode was enabled (Xcode 10.3). Though the generated rust lib do contain bitcode markers.